modules/core/src/main/scala/sbtHood/model/Decoders.scala

```diff
@@ -0,0 +1,12 @@
+package sbtHood.model
```

Comment:
`sbthood`?

Reply:
Actually just `hood` or `com.fortysevendeg.hood` to be accurate